movingpandas/trajectory.py

```diff
@@ -806,28 +920,51 @@ def _compute_angular_difference(self, row):
         else:
             return angular_difference(degrees1, degrees2)
 
-    def _compute_speed(self, row):
+    def _compute_speed(self, row, units=(DEFAULT_DISTANCE_UNITS, DEFAULT_TIME_UNITS)):
+        if units[1] == "default":
```

Comment:
It would be nice to use named tuples for the units so we can access the temporal unit with e.g. units.time and distance unit with units.dist instead of units[1] and units[0] (see e.g. https://realpython.com/python-namedtuple/)